quantum/rgblight_list.h

```diff
@@ -16,44 +16,46 @@
 #pragma once
 
 /*              RGB COLORS             */
-#define RGB_WHITE 0xFF, 0xFF, 0xFF
-#define RGB_RED 0xFF, 0x00, 0x00
-#define RGB_CORAL 0xFF, 0x7C, 0x4D
-#define RGB_ORANGE 0xFF, 0x80, 0x00
-#define RGB_GOLDENROD 0xD9, 0xA5, 0x21
-#define RGB_GOLD 0xFF, 0xD9, 0x00
-#define RGB_YELLOW 0xFF, 0xFF, 0x00
-#define RGB_CHARTREUSE 0x80, 0xFF, 0x00
-#define RGB_GREEN 0x00, 0xFF, 0x00
-#define RGB_SPRINGGREEN 0x00, 0xFF, 0x80
-#define RGB_TURQUOISE 0x47, 0x6E, 0x6A
-#define RGB_TEAL 0x00, 0x80, 0x80
-#define RGB_CYAN 0x00, 0xFF, 0xFF
-#define RGB_AZURE 0x99, 0xf5, 0xFF
-#define RGB_BLUE 0x00, 0x00, 0xFF
-#define RGB_PURPLE 0x7A, 0x00, 0xFF
-#define RGB_MAGENTA 0xFF, 0x00, 0xFF
-#define RGB_PINK 0xFF, 0x80, 0xBF
+#define RGB_OFF          0x00, 0x00, 0x00
+#define RGB_WHITE        0xFF, 0xFF, 0xFF
+#define RGB_RED          0xFF, 0x00, 0x00
+#define RGB_CORAL        0xFF, 0x7C, 0x4D
+#define RGB_ORANGE       0xFF, 0x80, 0x00
+#define RGB_GOLDENROD    0xD9, 0xA5, 0x21
+#define RGB_GOLD         0xFF, 0xD9, 0x00
+#define RGB_YELLOW       0xFF, 0xFF, 0x00
+#define RGB_CHARTREUSE   0x80, 0xFF, 0x00
+#define RGB_GREEN        0x00, 0xFF, 0x00
+#define RGB_SPRINGGREEN  0x00, 0xFF, 0x80
+#define RGB_TURQUOISE    0x47, 0x6E, 0x6A
+#define RGB_TEAL         0x00, 0x80, 0x80
+#define RGB_CYAN         0x00, 0xFF, 0xFF
+#define RGB_AZURE        0x99, 0xf5, 0xFF
+#define RGB_BLUE         0x00, 0x00, 0xFF
+#define RGB_PURPLE       0x7A, 0x00, 0xFF
+#define RGB_MAGENTA      0xFF, 0x00, 0xFF
+#define RGB_PINK         0xFF, 0x80, 0xBF
 
 /*            HSV COLORS            */
-#define HSV_WHITE 0, 0, 255
-#define HSV_RED 0, 255, 255
-#define HSV_CORAL 11, 176, 255
-#define HSV_ORANGE 28, 255, 255
-#define HSV_GOLDENROD 30, 218, 218
-#define HSV_GOLD 36, 255, 255
-#define HSV_YELLOW 43, 255, 255
-#define HSV_CHARTREUSE 64, 255, 255
-#define HSV_GREEN 85, 255, 255
-#define HSV_SPRINGGREEN 106, 255, 255
-#define HSV_TURQUOISE 123, 90, 112
-#define HSV_TEAL 128, 255, 128
-#define HSV_CYAN 128, 255, 255
-#define HSV_AZURE 132, 102, 255
-#define HSV_BLUE 170, 255, 255
-#define HSV_PURPLE 191, 255, 255
-#define HSV_MAGENTA 213, 255, 255
-#define HSV_PINK 234, 128, 255
+#define HSV_OFF            0,   0,   0
+#define HSV_WHITE          0,   0, 255
+#define HSV_RED            0, 255, 255
+#define HSV_CORAL         11, 176, 255
+#define HSV_ORANGE        28, 255, 255
+#define HSV_GOLDENROD     30, 218, 218
+#define HSV_GOLD          36, 255, 255
+#define HSV_YELLOW        43, 255, 255
+#define HSV_CHARTREUSE    64, 255, 255
+#define HSV_GREEN         85, 255, 255
+#define HSV_SPRINGGREEN  106, 255, 255
+#define HSV_TURQUOISE    123,  90, 112
+#define HSV_TEAL         128, 255, 128
+#define HSV_CYAN         128, 255, 255
+#define HSV_AZURE        132, 102, 255
+#define HSV_BLUE         170, 255, 255
+#define HSV_PURPLE       191, 255, 255
+#define HSV_MAGENTA      213, 255, 255
+#define HSV_PINK         234, 128, 255
```

Comment:
Could this change be reverted? 

Specifically, we want to keep changes to core files separate from user or even keyboard related changes.  This is especially important if we ever need to roll back a commit. 

If you want, you can submit this change as a separate PR. 

Also, this change would be reverted as soon as it was committed, since we run a re-format pass after merging.

Reply:
Will do